libs/pages/application/src/lib/ui/container/container.tsx

```diff
@@ -124,40 +125,39 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
   )
 
   const headerActions = (
-    <>
+    <div className="flex flex-col gap-3">
+      <div className="flex flex-row gap-2">
+        {environment && (
+          <Skeleton width={80} height={32} show={!environment?.mode}>
+            <EnvironmentMode size="xs" mode={environment.mode} />
```

Comment:
Sorry, I didn't see that on the previous PR

- Could you add the full name for the labels "dev" and "prod"?

- The height of each label by default should be 20px and not 26px 

- For this kind of component, we can remove the padding on top and bottom. Centered with flex-box, should be better and avoid strange behavior.

![Capture d’écran 2023-09-25 à 16 20 48](https://github.com/Qovery/console/assets/533928/619e9c3f-0603-4534-9ea3-501b4c3ef9c8)

Reply:
https://github.com/Qovery/console/pull/850/commits/7f616032c13c6b1448d5fff032779ae011dce49a